app/main/routes.py

```diff
@@ -74,6 +72,7 @@ def upload():
                 fund=Config.FUND_NAME,
             )
         else:
+            current_app.logger.info("Uncaught exception during upload")
```

Comment:
Bit confused when this scenario would come up? If there's an uncaught exception, would we not throw an exception also?

Reply:
Yep this is legacy logic - due to be fixed explicitly in an upcoming ticket https://dluhcdigital.atlassian.net/browse/SMD-229. I've removed this old template from being used for now though and replaced it with a standard 500 error.